src/todolist.js

```diff
@@ -0,0 +1,49 @@
+export default class ToDolist {
+  #toDo;
+
+  constructor() {
+    this.#toDo = [];
+  }
+
+  SetToDo(toDo) {
+    this.#toDo = toDo;
+  }
+
+  get ToDo() {
+    return this.#toDo;
+  }
+
+  reorder() {
+    for (let index = 0; index < this.#toDo.length; index += 1) {
+      this.#toDo[index].id = index;
+    }
+  }
+
+  // Methods
+  AddToDo(activity, doneBox) {
+    const id = this.#toDo.length;
```

Comment:
- [ ] I noticed that your index is the length of the array before the new task is pushed in so your index starts from zero. Please as demanded by project requirements, kindly make the property `index` set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5). You an achieve this by changing line `24` of your code to `const index = this.#toDo.length + 1;`